src/site/apt/index.apt.vm

```diff
@@ -448,7 +395,6 @@ Welcome to ${project.name}!
  * <<<custom.fluidoSkin.facebookLike.shareButton>>>: <<optional>>, if set to <<<true>>> <<Share Button>> will be included.
 
  * <<<custom.fluidoSkin.facebookLike.layout>>>: <<optional>> facebook layout, Can be one of:
-
```

Comment:
Are you certain about this line? Apt won't handle missing lines properly.

Reply:
thx - reverted